src/views/ValidatorItem.vue

```diff
@@ -3,8 +3,10 @@
     <div class="view-main__title-wrapper">
       <BackButton :text="'Validators'" />
       <h2 class="view-main__title">Validator</h2>
-      <div class="validator__address">
-        <p class="view-main__subtitle">{{ validator?.operatorAddress }}</p>
+      <div class="view-main__subtitle-wrapper">
+        <p class="view-main__subtitle fs-cut">
+          {{ validator?.operatorAddress }}
```

Comment:
what if operationAddress will not be there? you will display undefined?
handle this place -> you can create computed property and check there

Reply:
done